### ISSUE TITLE
New Releases?

### ISSUE THREAD

**Issue by derekwinters** [outsideris/citizen#55]
Is there a plan to tag a release for the latest changes and publish them to dockerhub? The latest release is 0.3.3 but there are a few new changes ( like #42 ) that I would need to be able to use this. It looks really promising but without version 12 support I won't be able to get very far.

**Comment by outsideris** [comment]
Supporting 0.12 is implemented in #51. 
And I will new release after merging #50 . I work on progress on my local because I fixed many codes after #50 .
I think supporting provider is important with terraform 0.14+. 

I will do my best. Thank you for your interest.

**Comment by derekwinters** [comment]
Thanks. I've been looking at the project more, and I was able to pull it down and run locally for some initial testing. I've made a few small changes, if I can get them stable enough I will send over a PR to see if you'd want to implement them as well. I added a few more properties from the hcl parser and I'm hoping to be able to add support for Azure Blob storage once I get more familiar with the project.

Thank you for this project, it looks incredibly promising.
